bokeh/io.py

```diff
@@ -609,3 +610,31 @@ def _remove_roots(subplots):
     for sub in subplots:
         if sub in doc.roots:
             doc.remove_root(sub)
+
+def server_cell(server, script):
```

Comment:
Happy to move it! 

This only needs to be used by ``show_app``so it can be made private wherever that goes. Where would you like ``show_app`` to end up?